src/Data/index.js

```diff
@@ -0,0 +1,24 @@
+const projects = [
+  {
```

Comment:
I think you should not have a folder just to have this file. Also, the name Data is not much descriptive